src/app/use-cases/validate-document.ts

```diff
@@ -45,12 +46,24 @@ export type ValidateDocumentPayload = GetDocumentPayload & {
   documentUrl: string;
 };
 
+const nextStatus = (
```

Comment:
this method should handle all cases (not only from DRAFT to WAIT_FOR_ISSUER),
you probably want to use a switch statement here.